<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MaterialCalculation</class>
 <widget class="QWidget" name="MaterialCalculation">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>492</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>201</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>33、24材料计算</string>
  </property>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>404</y>
     <width>183</width>
     <height>50</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>183</width>
     <height>50</height>
    </size>
   </property>
   <property name="title">
    <string>人数</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_13">
    <item row="0" column="1">
     <widget class="QRadioButton" name="radioButton_three">
      <property name="toolTipDuration">
       <number>3</number>
      </property>
      <property name="text">
       <string>3人</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QRadioButton" name="radioButton_two">
      <property name="toolTipDuration">
       <number>2</number>
      </property>
      <property name="text">
       <string>2人</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_calculation">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>460</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_close">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>460</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>191</width>
     <height>391</height>
    </rect>
   </property>
   <property name="title">
    <string>材料数量</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_15">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_LYJJ">
        <property name="text">
         <string>雷云结晶：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_LYJJ"/>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_FHJJ">
        <property name="text">
         <string>烦惑结晶：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_FHJJ"/>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QGridLayout" name="gridLayout_14">
      <item row="0" column="0">
       <widget class="QLabel" name="label_FHJJ_3">
        <property name="text">
         <string>天寿结晶：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_TSJJ"/>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QLabel" name="label_LYJJ_2">
        <property name="text">
         <string>南天魂石：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_NTHS"/>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="label_LYJJ_3">
        <property name="text">
         <string>冥幻魂石：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_MHHS"/>
      </item>
     </layout>
    </item>
    <item row="5" column="0">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="label_LYJJ_4">
        <property name="text">
         <string>雪影镜：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_XYJ"/>
      </item>
     </layout>
    </item>
    <item row="6" column="0">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <widget class="QLabel" name="label_LYJJ_5">
        <property name="text">
         <string>金牙镜：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_JYJ"/>
      </item>
     </layout>
    </item>
    <item row="7" column="0">
     <layout class="QGridLayout" name="gridLayout_9">
      <item row="0" column="0">
       <widget class="QLabel" name="label_LYJJ_8">
        <property name="text">
         <string>曜火镜：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_YHJ"/>
      </item>
     </layout>
    </item>
    <item row="8" column="0">
     <layout class="QGridLayout" name="gridLayout_7">
      <item row="0" column="0">
       <widget class="QLabel" name="label_LYJJ_6">
        <property name="text">
         <string>晨晖星石：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_CHXS"/>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <layout class="QGridLayout" name="gridLayout_8">
      <item row="0" column="0">
       <widget class="QLabel" name="label_LYJJ_7">
        <property name="text">
         <string>金雷星石：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_JLXS"/>
      </item>
     </layout>
    </item>
    <item row="10" column="0">
     <layout class="QGridLayout" name="gridLayout_10">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_YGXS"/>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_LYJJ_9">
        <property name="text">
         <string>月光星石：</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="11" column="0">
     <layout class="QGridLayout" name="gridLayout_11">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>其他备注：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
     </layout>
    </item>
    <item row="12" column="0">
     <layout class="QGridLayout" name="gridLayout_12">
      <item row="0" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>其他总价格：</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="doubleSpinBox_Other">
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_close</sender>
   <signal>clicked()</signal>
   <receiver>MaterialCalculation</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>150</x>
     <y>478</y>
    </hint>
    <hint type="destinationlabel">
     <x>102</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
